recipes/emsdk/all/conanfile.py

```diff
@@ -19,6 +19,18 @@ class EmSDKConan(ConanFile):
     license = "MIT"
     settings = "os", "arch", "compiler", "build_type"
 
+    options = {
+        "nodejs_version": [
+            "16.3.0",
+            "16.20.2",
+            "18.15.0"
+        ]
+    }
+
+    default_options = {
+        "nodejs_version": "16.3.0"
+    }
+
```

Comment:
```suggestion
```
Please, don't do that. The requirement should be package of the package ID already, adding it as an option is a good approach, you will can use use a recipe and override to use the version that you want.